verilog/analysis/verilog_filelist_test.cc

```diff
@@ -89,18 +69,18 @@ TEST(FileListTest, ParseSourceFileListFromCommandline) {
       "file3",
       "+define+macro6=a=b",
       "+incdir+../path/to/file4+./path/to/file5"};
-  auto parsed_file_list = ParseSourceFileListFromCommandline(cmdline);
-  ASSERT_TRUE(parsed_file_list.ok());
+  FileList result;
+  auto status = AppendFileListFromCommandline(cmdline, &result);
+  ASSERT_TRUE(status.ok());
```

Comment:
Maybe print the actual status received when it fails, something like
`ASSERT_TRUE(status.ok()) << "Got status: " << status.message();`

I can't remember of `ASSERT_OK` (and relatives) already do this.

Reply:
We don't have `ASSERT_OK()` unfortunately in the public gtest. Changing the asserts to print the status.